index.js

```diff
@@ -1,9 +1,50 @@
 'use strict';
 var path = require('path');
+var resolve = require('resolve');
+var Funnel = require('broccoli-funnel');
+var MergeTrees = require('broccoli-merge-trees');
 
 module.exports = {
   name: 'ember-cli-pretender',
 
+  init: function() {
+    this._super.init && this._super.init.apply(this, arguments);
+
+    this._pretenderPath = resolve.sync('pretender');
```

Comment:
Since these are not needed for non-build commands, can we move this to a helper method and ensure they are setup in `treeForVendor` / `included`?  I'd hate to do extra work during all commands that may not be building (ala `ember v`).